third_party/config/build/release.yaml

```diff
@@ -0,0 +1,350 @@
+kind: Image
+metadata:
+  name: gcs-fetcher
+  namespace: knative-build
+spec:
+  image: gcr.io/cloud-builders/gcs-fetcher
+---
+apiVersion: caching.internal.knative.dev/v1alpha1
+kind: Image
+metadata:
+  name: nop
+  namespace: knative-build
+spec:
+  image: gcr.io/knative-releases/github.com/knative/build/cmd/nop@sha256:b674b5c252deeae84b9e65d55f2da4ee63fbb6f123ebe481709e6f9c14faa4e1
+---
+apiVersion: v1
+data:
+  loglevel.controller: info
+  loglevel.creds-init: info
+  loglevel.git-init: info
+  loglevel.webhook: info
+  zap-logger-config: |
+    {
+      "level": "info",
+      "development": false,
+      "sampling": {
+        "initial": 100,
+        "thereafter": 100
+      },
+      "outputPaths": ["stdout"],
+      "errorOutputPaths": ["stderr"],
+      "encoding": "json",
+      "encoderConfig": {
+        "timeKey": "",
+        "levelKey": "level",
+        "nameKey": "logger",
+        "callerKey": "caller",
+        "messageKey": "msg",
+        "stacktraceKey": "stacktrace",
+        "lineEnding": "",
+        "levelEncoder": "",
+        "timeEncoder": "",
+        "durationEncoder": "",
+        "callerEncoder": ""
+      }
+    }
+kind: ConfigMap
+metadata:
+  name: config-logging
+  namespace: knative-build
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: build-controller
+  namespace: knative-build
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: build-controller
+    spec:
+      containers:
+      - args:
+        - -logtostderr
+        - -stderrthreshold
+        - INFO
+        - -creds-image
+        - gcr.io/knative-releases/github.com/knative/build/cmd/creds-init@sha256:770b411f241159d71210bdee7d51a608bc77ae1542dbc3e9ca9465316413b680
+        - -git-image
+        - gcr.io/knative-releases/github.com/knative/build/cmd/git-init@sha256:b5128bc1e7c476c1036f9a2c45d499bf6a0c132ef1baa75d1fee155ae23421a6
+        - -nop-image
+        - gcr.io/knative-releases/github.com/knative/build/cmd/nop@sha256:b674b5c252deeae84b9e65d55f2da4ee63fbb6f123ebe481709e6f9c14faa4e1
+        image: gcr.io/knative-releases/github.com/knative/build/cmd/controller@sha256:04fa3a155337c65db154083e3240ea3b309093f377536466a222b3e8d774ae08
+        name: build-controller
+        volumeMounts:
+        - mountPath: /etc/config-logging
+          name: config-logging
+      serviceAccountName: build-controller
+      volumes:
+      - configMap:
+          name: config-logging
+        name: config-logging
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: build-webhook
+  namespace: knative-build
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: build-webhook
+        role: build-webhook
+    spec:
+      containers:
+      - args:
+        - -logtostderr
+        - -stderrthreshold
+        - INFO
+        image: gcr.io/knative-releases/github.com/knative/build/cmd/webhook@sha256:4f6458cafbe29413bf5187ca560c0e68f004080f123afe32dcd2c8c6b27ed3d3
```

Comment:
We can depend on specific tag of image and use the https://github.com/grafeas/kritis/tree/master/cmd/kritis/kubectl/plugins/resolve plugin to resolve the tag.

Reply:
not sure how this would work, but sounds cool. We can refactor to change that?